tests/test_compress_assets_plugin.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding:utf-8 -*-
+
+import os
+
+import pytest
+
+from sigal.gallery import Gallery
+from sigal import init_plugins
+from sigal.plugins import compress_assets
+
+
+def make_gallery(settings, tmpdir, method):
+    settings['destination'] = str(tmpdir)
+    if "sigal.plugins.compress_assets" not in settings["plugins"]:
+        settings['plugins'] += ["sigal.plugins.compress_assets"]
```

Comment:
you should unregister the blinker signal at the end of the test to avoid side effects, something similar to https://github.com/saimn/sigal/commit/50072e2e2fa2f650dd1899b14aaaecb2dfe909ef#diff-56f7d7b4778bac73b6b655c02c8467aaR26

Reply:
Done with a fixture, inside the test itself. Feel free to copy it to conftest.py if you think it's useful.

Reply:
Looks good, thanks !